src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,51 @@
+import { FC } from 'react';
+import { Person } from '../../types';
+import { PersonInfo } from '../PersonInfo/PersonInfo';
+import { getParent } from '../../helpers';
+
+type Props = {
+  people: Person[],
+};
+
+export const PeopleTable: FC<Props> = ({ people }) => (
+  <table
+    data-cy="peopleTable"
+    className="table is-striped is-hoverable is-narrow is-fullwidth"
+  >
+    <thead>
+      <tr>
+        <th>Name</th>
+        <th>Sex</th>
+        <th>Born</th>
+        <th>Died</th>
+        <th>Mother</th>
+        <th>Father</th>
+      </tr>
+    </thead>
+
+    <tbody>
+      {people.map(person => {
+        const {
+          slug,
+          motherName,
+          fatherName,
+        } = person;
+        const mother = getParent(people, motherName);
+        const father = getParent(people, fatherName);
```

Comment:
Let's prepare array to work with it after we receive data from server instead of doing it on every component render